src/ray/protobuf/common.proto

```diff
@@ -143,6 +143,8 @@ message Bundle {
   }
   BundleIdentifier bundle_id = 1;
   map<string, double> unit_resources = 2;
+  // Whether the bundle is placed.
+  bool is_placed = 3;
```

Comment:
Do we really need this? It seems we can infer it based on whether the bundle exists on some node.

Reply:
I use `node_id` field instead of `is_placed` field. When GCS is restarted, we can use `node_id` to restore the bundle location.